### PR TITLE
Create Serve Pipeline REP

### PR DESCRIPTION
**Pull request by jiaodong**
This .md is our REP-001 to setup an example of our ray community enhancement process. It follows our REP guideline with detailed examples of Serve Pipeline Beta launch.

Related Issues: #2

### CODE REVIEW

File: reps/serve_pipeline.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+## Stewardship
+### Required Reviewers
+The proposal will be open to the public, but please suggest a few experience Ray contributors in this technical domain whose comments will help this proposal. Ideally, the list should include Ray committers.
+
+@ericl, @edoakes, @simon-mo, @jiaodong
+
+### Shepherd of the Proposal (should be a senior committer)
+To make the review process more productive, the owner of each proposal should identify a **shepherd** (should be a senior Ray committer). The shepherd is responsible for working with the owner and making sure the proposal is in good shape (with necessary information) before marking it as ready for broader review.
+
+@ericl
```
Comment (ericl) [inline reps/serve_pipeline.md:27]: @zhe-thoughts should the shepherd just be the person reviewing / merging the PR into this repo? If so, we don't need to list it explicitly in the REP.

File: reps/serve_pipeline.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+
```
Comment (ericl) [inline reps/serve_pipeline.md:1]: Could we add a date to this file name? Such as YYYY-MM-DD-title.md, so `2022-03-08-serve_pipeline.md`

That will ensure a sensible sort order in the github listing.

**Comment by ericl** [comment]
Content LGTM!

File: reps/serve_pipeline.md (hunk 1 above)
Comment (zhe-thoughts) [reply reps/serve_pipeline.md]: @ericl Yes I think the workflow can be that the proposer designate the shepherd on the `Reviewers` field in the PR, and the shepherd should be the one to merge it. But how do people find the info once the REP is a committed .md file? As long as they can find out who's the shepherd, I'm OK with it

Comment (ericl) [reply reps/serve_pipeline.md]: Could always look through the merged PRs list (i.e., this thread).

**Review by ericl — APPROVED** [review]
I think this is good to merge. Want to also update the PR description?

**Comment by jiaodong** [comment]
updated -- merging now.

**Comment by jiaodong** [comment]
@ericl should i keep the issue open for tracking and discussions, or just the .md file in repo is enough ?

**Comment by zhe-thoughts** [comment]
@jiaodong @ericl Thanks! Yup I think keeping the issue open for comments is good

**Pull request merged**